sqlglot/dialects/dialect.py

```diff
@@ -510,8 +510,28 @@ def get_or_raise(cls, dialect: DialectType) -> Dialect:
             return dialect
         if isinstance(dialect, str):
             try:
-                dialect_name, *kv_pairs = dialect.split(",")
-                kwargs = {k.strip(): v.strip() for k, v in (kv.split("=") for kv in kv_pairs)}
+                dialect_name, *kv_strings = dialect.split(",")
+                kv_pairs = (kv.split("=") for kv in kv_strings)
+                kwargs = {}
+                for pair in kv_pairs:
+                    key = pair[0].strip()
+                    value: t.Union[bool | str | None] = None
+
+                    if len(pair) == 1 and not key.lower() == "normalization_strategy":
+                        # Default initialize standalone settings to True except normalization strategy
+                        value = True
```

Comment:
Let's not check for `normalization_strategy`, we will fail in the constructor anyway and point out the user error that way.

Reply:
This part comes before the dialect constructor right? So we can change this condition to the following one

```
if len(pair) == 1 :
  if key.lower() == "normalization_strategy":
    # throw AttributEerror
    
   # Default initialize standalone settings to True except normalization strategy
   value = True
```